src/main/java/appeng/util/item/AEItemStack.java

```diff
@@ -120,7 +123,7 @@ public static IAEItemStack loadItemStackFromPacket( final ByteBuf data ) throws
 			return null;
 		}
 
-		final AEItemStack item = AEItemStack.create( itemstack );
+		final AEItemStack item = AEItemStack.fromItemStack( itemstack );
 		item.setStackSize( stackSize );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "item" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2259)